dagmc_stats.py

```diff
@@ -72,3 +75,24 @@ def get_entityset_ranges(my_core, meshset, geom_dim):
         entityset_ranges[set_type] = my_core.get_entities_by_type_and_tag(meshset, types.MBENTITYSET, geom_dim, 
                                                                           [dimension])
     return entityset_ranges
+
+
+def get_surfaces_per_volume(my_core, entityset_ranges):
+    """
+    Get the number of surfaces that each volume in a given file contains
+
+    inputs
+    ------
+    my_core : a MOAB core instance
+    entity_ranges : a dictionary of the entityset ranges of each tag in a file
+
+    outputs
+    -------
+    freqs : a list of the number of surfaces each volume in the file contains
+
+    """
+
+    s_p_v = []
```

Comment:
why not declare it as an `np.array` right away?
```suggestion
    s_p_v = np.array()
```

Reply:
using `np.array()` with no arguments raises an error

Reply:
But I was able to work around it